pkg/pub_package_reader/lib/src/file_names.dart

```diff
@@ -2,25 +2,15 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// Returns common file name candidates for [base] (specified without any extension).
-List<String> textFileNameCandidates(String base) {
-  return <String>[
-    base,
-    '$base.md',
-    '$base.markdown',
-    '$base.mkdown',
-    '$base.txt',
-  ];
-}
-
-final List<String> changelogFileNames = textFileNameCandidates('changelog');
```

Comment:
Does this mean we accepted all sorts of names for the changelog before?

Will this break packages? Or is there another place where we require the changelog to be CHANGELOG.md?

Reply:
> Does this mean we accepted all sorts of names for the changelog before?

possibly `ChaNgeLoG.mD` was accepted

> Will this break packages?

pub.dev won't display the above file as changelog for new uploads or when we re-extract assets

> Or is there another place where we require the changelog to be CHANGELOG.md?

pana (needs to be updated too)